content/en/apps/examples/training.md

```diff
@@ -38,18 +38,36 @@ The CHT’s onboarding and training capabilities offer a remote way to provide e
 
 ## Workflow Examples
 
-![workflow.png](workflow1.png)
+### Remote Login by App
+
+When creating users, the admin has the option to send a user their credentials via SMS using a magic link. This generates a new, random and complex password with a 24-hour expiry. 
```

Comment:
may be helpful to add a link to the "magic link" - not sure everyone will know what it is

Reply:
good call!